operator_manifest/operator.py

```diff
@@ -615,7 +649,7 @@ def _get_csvs(cls, yaml_files, **kwargs):
         for f in yaml_files:
             try:
                 yield OperatorCSV.from_file(f, **kwargs)
-            except NotOperatorCSV:
+            except (NotOperatorCSV, CSVValidationError):
```

Comment:
This seems suspicious to me. Do we want to skip CSV instead of raising an error?

Reply:
after discussion with martin, I removed this error altogether as it was not raised anywhere.